cli/pkg/workspace/plugin/use.go

```diff
@@ -127,12 +127,12 @@ func (o *UseWorkspaceOptions) Run(ctx context.Context) (err error) {
 		name = "~" + strings.TrimPrefix(name, home)
 	}
 	name = strings.ReplaceAll(name, "/", ":")
-	if name == core.RootCluster.String() || strings.HasPrefix(name, core.RootCluster.String()+":") {
+	if name == core.RootCluster.String() || strings.HasPrefix(name, core.RootCluster.String()+":") && !o.ShortWorkspaceOutput {
 		// LEGACY(mjudeikis): Remove once everybody gets used to this
 		name = ":" + name
 		fmt.Fprintf(o.ErrOut, "Note: Using 'root:' to define an absolute path is no longer supported. Instead, use ':root' to specify an absolute path.\n")
 	}
```

Comment:
This has been here for over a year, this could also be removed.

Reply:
We probably haven't done a great job updating documentation around this, so I suppose we should maybe review that before pulling the plug.